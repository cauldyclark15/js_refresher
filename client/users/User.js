import React from 'react'
import PropTypes from 'prop-types'

const User = ({ name, phone, website }) => {
  return (
    <div className='post'>
      <div className='padding'>
        <div className='media'>
          <div className='media-body'>
            <h5 className='post-heading'>{name}</h5>
            <p className='post-info'>contact: {phone}</p>
            <p className='post-info'>website: {website}</p>
          </div>
        </div>
      </div>
    </div>
  )
}

User.propTypes = {
  name: PropTypes.string,
  phone: PropTypes.string,
  website: PropTypes.string
}

export default User
