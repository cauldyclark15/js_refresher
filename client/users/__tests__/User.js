/* eslint-env node, jest */
import React from 'react'
import User from '../User'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

test('render user component', function () {
  var component = shallow(<User name='jc' phone='1234567' website='http://www.sven.com' />)
  var tree = shallowToJson(component)
  expect(tree).toMatchSnapshot()
})
