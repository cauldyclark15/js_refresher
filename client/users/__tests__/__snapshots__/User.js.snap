// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render user component 1`] = `
<div
  className="post"
>
  <div
    className="padding"
  >
    <div
      className="media"
    >
      <div
        className="media-body"
      >
        <h5
          className="post-heading"
        >
          jc
        </h5>
        <p
          className="post-info"
        >
          contact: 
          1234567
        </p>
        <p
          className="post-info"
        >
          website: 
          http://www.sven.com
        </p>
      </div>
    </div>
  </div>
</div>
`;
